frontend/src/app/globe-ngb/datepicker-container.component.ts

```diff
@@ -0,0 +1,27 @@
+/* tslint:disable:component-selector */
+import { Component, ContentChild } from '@angular/core';
+import { NgbInputDatepicker } from '@ng-bootstrap/ng-bootstrap';
+
+/**
+ * Component used to simplify the markup of a datepicker in a popup. It wraps the input which has the
+ * ngbDatepicker directive, and prepends the toggle button to it.
+ *
+ * Example usage:
+ *
+ * <div class="col-sm-9 input-group" gl-datepicker-container>
+ *   <input class="form-control" formControlName="date" ngbDatepicker />
+ * </div>
+ */
+@Component({
+  selector: '[gl-datepicker-container]',
```

Comment:
nitpicking: I find it less intuitive than if it was a "classic" component when reading the other templates using it. You picked an attribute selector because it was less changes?